src/components/rangeslider/draw.js

```diff
@@ -376,6 +376,7 @@ function drawRangePlot(rangeSlider, gd, axisOpts, opts) {
         };
 
         mockFigure.layout[oppAxisName] = {
+            type: oppAxisOpts.type,
```

Comment:
Before, we were strictly relying on the axis-auto-type routines in `Plots.supplyDefaults` to determine the range plot y-axis type - which obviously fails when a overrides the y axis type (e.g. in the case of log y axes as presented in https://github.com/plotly/plotly.js/issues/1470)